Rubberduck.VBEEditor/WindowsApi/VbeAttachableSubclass.cs

```diff
@@ -47,20 +47,24 @@ public bool HasValidVbeObject
 
                 try
                 {
-                    if (Marshal.GetIUnknownForObject(_target.Target) != IntPtr.Zero)
+                    var unmanaged = Marshal.GetIUnknownForObject(_target.Target);
+                    if (unmanaged != IntPtr.Zero)
                     {
+                        try
+                        {
+                            Marshal.Release(unmanaged);
+                        }
+                        catch (COMException)
+                        {
+                            // this should be "safe" to hold the reference - it's not like it should go anywhere now... :-/
+                            SubclassLogger.Warn($"{ GetType().Name } failed to dispose of a held { typeof(T).Name } reference.");
```

Comment:
The log message doesn't actually reflect what really just happened. Probably should be `...failed to release unmanaged pointer {unmanaged} of a held....` or something like that.

Reply:
It isn't attempting to release the *pointer*, `Marshal.Release` attempts to call `.Release` on the `IUnknown` pointed to by the pointer in order to decrement its reference count. Documentation is [here](https://msdn.microsoft.com/en-us/library/system.runtime.interopservices.marshal.release(v=vs.110).aspx).

Reply:
...although I can see how "dispose of" implies `IDisposable`.  What about:

`failed to decrement reference count of a held { typeof(T).Name } reference`

Reply:
Better. 👍